create_box/main.py

```diff
@@ -1,5 +1,13 @@
 def create_box(height, width, character):
-    pass
+    height = int(height)
+    width = int(width)
+    text = ''
+    if height >= 1 and width >=1:
+        for i in range(0, height):
+            text += character*width + '\n'
```

Comment:
Leave a space for better readability here as well. text += character * width + '\n'